build_verify.sh

```diff
@@ -0,0 +1,48 @@
+#/usr/bin/env bash 
+
+
+fails=""
+
+# Invoke gofmt on the current directory.
+# -l flag causes it to only list files
+GOFMT_RESULTS=`gofmt -l .`
+if [ "$GOFMT_RESULTS" ]
```

Comment:
let's put `] then` into a single line. (i think that's more idiomatic shell scripting)